<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>258</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>264</width>
    <height>258</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QSpinBox" name="minutesBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1440</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="daysBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Select project title to remove</string>
   </property>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QComboBox" name="projectsList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="repeatLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Set repeat each</string>
   </property>
  </widget>
  <widget class="QLabel" name="historyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Keep history for</string>
   </property>
  </widget>
  <widget class="QLabel" name="minutesLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>minutes</string>
   </property>
  </widget>
  <widget class="QLabel" name="dayLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>day(s)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
